docs/dev_guide/pr_review_procedure.rst

```diff
@@ -0,0 +1,54 @@
+.. _pr_review:
+
+This document describes the procedure that the people in the sunpy-maintainters group must follow when merging a pull request.
+If in any doubt about merging a pull request, the correct course of action is to contact the Lead Developer and ask.
+
+Fundamental Components of a Pull Request
+========================================
+
+Each pull request *must* meet the following criteria before it is considered for merge:
+
+-  The code must be PEP 8 compliant and meet the ill-defined SunPy quality standards.
+   We have these in the :ref:`coding-standards` page.
+-  The PR must contain a CHANGELOG entry if it changes the behavior of any code.
+-  The test coverage **ABSOLUTELY** must not decrease, and for new features should be at or very close to 100%.
```

Comment:
I know I wrote this but we can probably tone down the absolutely here :wink: maybe just normal text!